src/Facebook/Facebook.php

```diff
@@ -586,4 +589,72 @@ public function videoToUpload($pathToFile)
     {
         return new FacebookVideo($pathToFile);
     }
+
+    /**
+     * Upload video
+     *
+     * @param int $target
+     * @param string $pathToFile
+     * @param string $accessToken
+     * @param int $maxTransferTries
+     *
+     * @return array
+     *
+     * @throws FacebookSDKException
+     */
+    public function uploadVideo($target, $pathToFile, $accessToken, $maxTransferTries = 5)
```

Comment:
The chunked video feature was added in 2.3 but I think it's back-versioned to work with all versions of Graph. I can't find the doc reference to that and I haven't tested it myself but I feel like I remember hearing that at F8 this year. :)

On that point though - I think we should allow these types of checks to happen on the Graph API side so we don't have to keep up with all these little "gotchas" on the SDK end as the Graph API changes. Thoughts?